test/fuzzer/pedro/floating_point_conversion_overflow.test

```diff
@@ -14,6 +14,8 @@ SELECT 0::${float_type}::${type}
 
 statement error
 SELECT (-1)::${float_type}::${type}
+----
+can't be cast because the value is out of range for the destination type
```

Comment:
Here too, could we get `conversion error`?

Reply:
Sadly no, the curse of having multiple `${variable}` in a statement, the errors look like this:
```
Conversion Error: Type FLOAT with value -1.0 can't be cast because the value is out of range for the destination type UINT8
```

We can't find a better pattern in that